demisto_sdk/commands/validate/file_validator.py

```diff
@@ -393,13 +394,15 @@ def validate_added_files(self, added_files):  # noqa: C901
                 if self.id_set_validator.is_file_has_used_id(file_path):
                     self._is_valid = False
 
-            elif re.match(PLAYBOOK_REGEX, file_path, re.IGNORECASE):
+            elif re.match(PLAYBOOK_REGEX, file_path, re.IGNORECASE) or file_type == 'playbook':
                 playbook_validator = PlaybookValidator(structure_validator)
                 if not playbook_validator.is_valid_playbook():
                     self._is_valid = False
 
-            elif checked_type(file_path, YML_INTEGRATION_REGEXES):
+            elif checked_type(file_path, YML_INTEGRATION_REGEXES) or file_type == 'integration':
                 image_validator = ImageValidator(file_path)
+                # if file_type(non git path) the image is not in a separate path
```

Comment:
not sure this is always true

Reply:
the -p is to test a single file. usually one downloaded from demisto.
what do you think should be done here?